src/php/structs.rs

```diff
@@ -239,3 +238,54 @@ impl PhpBinary {
         }
     }
 }
+
+
+//
+//
+//
+//
+//
+//
+//
+//
+
+
+#[derive(Hash, Eq, PartialEq, Debug, Serialize, SerdeDeserialize)]
+pub(crate) struct ProcessInfo {
```

Comment:
I think it could be more self-explanatory to rename this struct to `ServerInfo` instead, WDYT?

Reply:
Absolutly. I have renamed the struct.